3600/1/parser.rb

```diff
@@ -0,0 +1,82 @@
+require 'nokogiri'
+require './gems_stats'
+require 'open-uri'
+require 'json'
+
+# :reek:InstanceVariableAssumption
+class Parser
+  attr_reader :gem_name
+
+  def initialize(gem_name:)
+    @gem_name = gem_name
+  end
+
+  def repo_url
+    return @repo_url if @repo_url
+    json_str = open("https://rubygems.org/api/v1/gems/#{gem_name}.json").read
+    @repo_url = JSON(json_str)['source_code_uri']
+  end
+
+  def repo_dependents_url
```

Comment:
rename, if u remove repo_ you will still understand what this method about

Reply:
repo prefix delited